tests/test_clock.py

```diff
@@ -43,46 +43,42 @@ def test_timer_duration_exceeded(self):
         duration = datetime.timedelta(hours=3 * 24)
         self.mock_clock.now.side_effect = [
             datetime.datetime(
-                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
-            datetime.datetime(
-                2016, 7, 27, 10, 51, 9, 928000, tzinfo=pytz.utc)
+                2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc), datetime.datetime(
+                    2000, 1, 5, 0, 0, 0, 0, tzinfo=pytz.utc)
         ]
```

Comment:
Oh yeah, this is wonky. We can change the right bracket to:

```python
]  # yapf: disable
```

And then YAPF won't touch the line. Worth it when it gets it this bad. But maybe moot based on my comment below.